group-coordinator/src/test/java/org/apache/kafka/coordinator/group/RecordHelpersTest.java

```diff
@@ -764,4 +813,39 @@ public void testNewOffsetCommitTombstoneRecord() {
         Record record = RecordHelpers.newOffsetCommitTombstoneRecord("group-id", "foo", 1);
         assertEquals(expectedRecord, record);
     }
+
+    // For the purpose of testing let:
+    //  a) Number of replicas for each partition = 2
+    //  b) Number of racks available in the cluster = 4
```

Comment:
Should we remove this? It does not make sense here. Or was it supposed to be somewhere else?

Reply:
Its applicable to both the mkMapOfPartitionRacks and mkListOfPartitionRacks methods to show why there's two rackIds in each set and why the mod is 4. I got confused when I saw the old tests and didn't understand why a certain calculation was made and what the assumptions behind it were. I initially added it to the javadoc for the first method but realised it probably doesn't belong there. I'm not sure if I should place it inside the method for both or where the correct placement would be

Reply:
The javadoc is definitely a better place than this.

Reply:
Okay I will add it to the javadoc